core/src/tpu.rs

```diff
@@ -163,11 +166,15 @@ impl Tpu {
             staked_nodes.clone(),
             MAX_STAKED_CONNECTIONS,
             MAX_UNSTAKED_CONNECTIONS,
+            MAX_STREAMS_PER_MS_TPU,
             DEFAULT_WAIT_FOR_CHUNK_TIMEOUT,
             tpu_coalesce,
         )
         .unwrap();
 
+        // 5K PPS
+        const MAX_STREAMS_PER_MS_TPU_FORWARD: u64 = 5;
```

Comment:
How did we come up with this constant? Did we do any testing on the perf impact?

Reply:
It was used in our testing on `3gG` -- I tested using bench-tps -- no observable difference probably because it uses little forward. But I revert this change to later keep using the current values.